site/themes/citra-bs-theme/layouts/shortcodes/mp4.html

```diff
@@ -0,0 +1,15 @@
+<figure>
+   <div style="position: relative; padding-bottom: 56.25%; padding-top: 30px; height: 0; overflow: hidden;">
+      <video preload="auto" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;" controls>
+         <source src="{{ .Get "src" }}" type="video/mp4">
+         Your browser doesn't support mp4 video. :(
```

Comment:
`:(` feels a bit.. odd.

Reply:
This is just copying the other shortcodes 🤷🏻

Reply:
Keeping the sad face for parity with the other shortcodes, if you want those changed in general I can do, but I won't be doing that with this PR.